main.py

```diff
@@ -269,7 +269,7 @@ def workers_edit(worker_id):
                 Worker.email: request.form["email"],
                 Worker.phone: request.form["phone"],
                 Worker.notes: request.form["notes"],
-                Worker.active: request.form["active"],
+                Worker.active: bool(request.form.get("active")),
```

Comment:
Gotta change from request.form to request.form.get since the latter returns None when the field has no value but the former throws a key error.